src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public void returnInfo() {
```

Comment:
The method name `returnInfo` is not appropriate as per Java naming conventions. It should be a verb or a verb phrase, for example, `printInfo` would be more suitable.